src/renderer/components/NodeGridElement.tsx

```diff
@@ -85,7 +76,7 @@ const NodeGridElement: React.FC<NodeGridElementProps> = ({ node, temporalEpoch,
 
     return (
         <button
-            title={showOperationNames && shouldShowLabel ? node.opName : ''}
+            title={showOpNames && shouldShowLabel ? node.opName : ''}
```

Comment:
nit: prefer no name shortening. `showOperationNames` is perfectly fine and more readable